### ISSUE TITLE
IO wait time 

### ISSUE THREAD

**Issue by gvdongen** [google/cadvisor#2760]
I am trying to determine the amount of time an application is waiting on IO. 
Disk io wait and FS io time are always empty for my application. What can be cause of this?
I use cadvisor 0.32 and a rather old Docker version (1.10). 

Is another way of approximating this by subtracting cpu_system and cpu_user from cpu_total?

Thank you!

**Comment by iwankgb** [comment]
Can you provide any logs?

**Comment by dims** [comment]
Closing this issue as it has been open since before January 1, 2024. If this is still relevant, please feel free to reopen or create a new issue.
